sources/metrics/Stride.Metrics.ServerApp/Controllers/Api/ActivityMetricsController.cs

```diff
@@ -0,0 +1,237 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+using Stride.Metrics.ServerApp.Data;
+using Stride.Metrics.ServerApp.Dtos;
+using Stride.Metrics.ServerApp.Dtos.Agregate;
+using Stride.Metrics.ServerApp.Extensions;
+using Stride.Metrics.ServerApp.Helpers;
+
+namespace Stride.Metrics.ServerApp.Controllers.Api;
+///<summary>Activity Install related eps</summary>
+[ApiController()]
+[Route("api")]
+public class ActivityMetricsController
+{
+    private readonly MetricDbContext _metricDbContext;
+    ///<summary>Activity Install related eps</summary>
+    public ActivityMetricsController(MetricDbContext metricDbContext)
+    {
+        _metricDbContext = metricDbContext;
+    }
+
+    /// <summary>
+    /// Gets
+    /// </summary>
+    /// <returns>uzs.</returns>
+
+    [HttpGet("high-usage")]
+    public IEnumerable<AggregationPerMonth> GetHighUsage()
+    {
+        var editorAppId = _metricDbContext.GetApplicationId(CommonApps.StrideEditorAppId.Guid);
+
+        var result = _metricDbContext.GetHighUsage(editorAppId);
+
+        return result;
+    }
+
+    /// <summary>
+    /// Gets the usage count per version in the last 30 days.
+    /// </summary>
+    /// <returns>The usage count per version in the last 30 days.</returns>
+
+    [HttpGet("projects-users")]
+    public IEnumerable<ProjectsUsersAggregation> GetProjectsUsers()
+    {
+        var result = _metricDbContext.MetricEvents
+            .Where(a => a.MetricId == 19)
```

Comment:
Can you put that 19 in a names constant and explain which metric it represents (like `const int XxxMetricId = 19;`)?